stratum/hal/lib/barefoot/bf_pipeline_utils.cc

```diff
@@ -32,6 +32,7 @@ bool IsArchive(const std::string& archive) {
   auto cleanup = gtl::MakeCleanup([&a]() { archive_read_free(a); });
   archive_read_support_filter_bzip2(a);
   archive_read_support_filter_gzip(a);
+  archive_read_support_filter_xz(a);
```

Comment:
I'm getting linker errors now:
```
libarchive/archive_read_support_filter_xz.c:6747: error: undefined reference to 'lzma_stream_decoder'
libarchive/archive_read_support_filter_xz.c:6747: error: undefined reference to 'lzma_alone_decoder'
libarchive/archive_read_support_filter_xz.c:7620: error: undefined reference to 'lzma_properties_decode'
libarchive/archive_read_support_filter_xz.c:27810: error: undefined reference to 'lzma_raw_decoder'
external/com_github_libarchive_libarchive/local-install/lib/libarchive.a(archive_read_support_filter_xz.o):archive_read_support_filter_xz.c:function xz_filter_read: error: undefined reference to 'lzma_code'
external/com_github_libarchive_libarchive/local-install/lib/libarchive.a(archive_read_support_filter_xz.o):archive_read_support_filter_xz.c:function xz_filter_read: error: undefined reference to 'lzma_crc32'
external/com_github_libarchive_libarchive/local-install/lib/libarchive.a(archive_read_support_filter_xz.o):archive_read_support_filter_xz.c:function xz_filter_close: error: undefined reference to 'lzma_end'
```

Either we add `-llzma` to the libarchive linkopts:
```
    linkopts = [
        "-lbz2",
        # "-llzma"
    ],
```

Or drop support for everything but gzip.

Reply:
I originally removed the xz format because of linking errors, but somehow it crept  back in. Let's remove the xz lines and it should be okay (bz2 is linked statically).

Reply:
whoops now we have a commit that fixes the link error.